### DISCUSSION TITLE
Bugs or misconfiguration?

### DISCUSSION THREAD

**Discussion by zenminimalist** [plausible/analytics#3385] [Self-Hosted Support] [answered]
I'm currently seeing strange behaviors in my setup. I'm not sure if I've misconfigured something, but they do seem like bugs to be. 

**1. Filters not working / displaying data**

After fully importing data from approx. 5 years and selecting a filter e.g. County = USA, no data is shown in any of the four fields. Here's a gif illustrating this behaviors.

![CleanShot 2023-09-29 at 00 52 50](https://github.com/plausible/analytics/assets/7798503/fd05b8f5-9249-4e3e-b828-5123434c0eb9)

**2. Maxmind GeoIP data not being used**

Now, I'm not sure if the Maxmind IP has to download data first, which would be in one week (according to the conv file) before it shows "Maxmind" data in the map or if it's using the data, but just not showing the correct copyright? 

![CleanShot 2023-09-29 at 00 53 53@2x](https://github.com/plausible/analytics/assets/7798503/a4e03928-1257-421b-8813-540438ac1bf8)

My relevant conv files are, just to rule out any stupid mistakes I've made:

**My complete docker-compose file**
<details>
version: "3.3"
services:
  mail:
    image: bytemark/smtp
    container_name: plausible-mail
    hostname: plausible-mail
    restart: always

  plausible_db:
    # supported versions are 12, 13, and 14
    image: postgres:14-alpine
    container_name: plausible-db
    restart: always
    volumes:
      - db-data:/var/lib/postgresql/data
    environment:
      - POSTGRES_PASSWORD=DID NOT CHANGE

  plausible_events_db:
    image: clickhouse/clickhouse-server:23.3.7.5-alpine
    container_name: plausible-clickhouse-db
    restart: always
    volumes:
      - event-data:/var/lib/clickhouse
      - ./clickhouse/clickhouse-config.xml:/etc/clickhouse-server/config.d/logging.xml:ro
      - ./clickhouse/clickhouse-user-config.xml:/etc/clickhouse-server/users.d/logging.xml:ro
    ulimits:
      nofile:
        soft: 262144
        hard: 262144

  plausible:
    image: plausible/analytics:v2.0
    container_name: plausible
    restart: always
    command: sh -c "sleep 10 && /entrypoint.sh db createdb && /entrypoint.sh db migrate && /entrypoint.sh run"
    depends_on:
      - plausible_db
      - plausible_events_db
      - mail
	- geoid
    ports:
      - 127.0.0.1:8000:8000
    env_file:
      - plausible-conf.env

  geoip:
    image: maxmindinc/geoipupdate
	container_name: geoip-maxmind
    environment:
      - GEOIPUPDATE_EDITION_IDS=GeoLite2-Country
      - GEOIPUPDATE_FREQUENCY=168 # update every 7 days
    env_file:
      - geoip-conf.env
    volumes:
      - geoip:/usr/share/GeoIP
    restart: always

volumes:
  db-data:
    driver: local
  event-data:
    driver: local
  geoip:
    driver: local
</details>

**My plausible-conf.env file**
<details>
BASE_URL=[https://pa.domain.com]

SECRET_KEY_BASE=XXXXXXXXXXX
ENABLE_EMAIL_VERIFICATION=false
DISABLE_REGISTRATION=true 
RELEASE_DISTRIBUTION=none
</details>

**My geoip-conf.env file**
<details>
GEOIPUPDATE_ACCOUNT_ID=007

GEOIPUPDATE_LICENSE_KEY=XXXXXX
</details>

**3. Full screen view on exits page causes "circle of death"**
If I click on every of the four boxes detail, button, I get a list of more detailed result. So far, so good. However, if I go to details on the "Exit Pages" I get a spinning wheel of LSD. 

https://github.com/plausible/analytics/assets/7798503/2f688ad5-0c89-4f67-82cc-820f0a6da6dc

**Comment by Daniel15** [comment]
> Maxmind GeoIP data not being used

You're missing the Maxmind license key in your plausible-conf.env file. The license keys for the GeoLite databases are free, you just need to sign up on MaxMind's site here: https://www.maxmind.com/en/geolite2/signup. This is documented here: https://plausible.io/docs/self-hosting-configuration (search for MAXMIND_LICENSE_KEY)

**Reply by zenminimalist** [reply]
>You're missing the Maxmind license key in your plausible-conf.env file.

So it also goes into the plausible-conf.env file, not just the geoip-conf.env. I thought it just needs to be in the geoip-conf.env file.

**Comment by ruslandoga** [comment, marked as answer]
Answered here https://github.com/plausible/analytics/discussions/3447#discussioncomment-7354789
